src/store/actions/loginAction.js

```diff
@@ -0,0 +1,33 @@
+export const FETCHING_USER_LOGIN = "FETCHING_USER_LOGIN"
+export const FETCHING_USER_LOGIN_SUCCESS = "FETCHING_USER_LOGIN_SUCCESS"
+export const FETCHING_USER_LOGIN_FAIL = "FETCHING_USER_LOGIN_FAIL"
+export const USER_LOGOUT = "USER_LOGOUT"
+
+export const loginAction = {
+    login,
+    logout
+};
+
+// LOGIN
+function login(username, password) {
+    return dispatch => {
+        dispatch({ type: FETCHING_USER_LOGIN });
+                (username, password)
+                    .then(res => {
+                        console.log(res)
+                            dispatch({ type: FETCHING_USER_LOGIN_SUCCESS });
+                            //history.push('/');
```

Comment:
found another line that could be deleted.

Reply:
deleted